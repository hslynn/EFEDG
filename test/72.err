Running /share/soft/openmpi-3.0.0_gcc/bin/mpirun.openmpi -hostfile /share/home/hslynn/.lsbatch/1546156566.208189.hostfile ./simplest
mpirun.openmpi: Forwarding signal 18 to job
