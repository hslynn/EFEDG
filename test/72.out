Sender: LSF System <lsfadmin@c6u24n02>
Subject: Job 208189: <72> in cluster <hpc.lsec.cc.ac.cn> Exited

Job <72> was submitted from host <ln02> by user <hslynn> in cluster <hpc.lsec.cc.ac.cn>.
Job was executed on host(s) <36*c6u24n02>, in queue <batch>, as user <hslynn> in cluster <hpc.lsec.cc.ac.cn>.
                            <36*c2u10n03>
</share/home/hslynn> was used as the home directory.
</share/home/hslynn/Einstein_DG/test> was used as the working directory.
Started at Sun Dec 30 15:59:47 2018
Results reported on Sun Dec 30 16:15:56 2018

Your job looked like:

------------------------------------------------------------
# LSBATCH: User input
mpijob -t openmpi ./simplest
------------------------------------------------------------

TERM_OWNER: job killed by owner.
Exited with signal termination: Killed.

Resource usage summary:

    CPU time :                                   542.24 sec.
    Max Memory :                                 1415 MB
    Average Memory :                             1318.08 MB
    Total Requested Memory :                     -
    Delta Memory :                               -
    Max Processes :                              37
    Max Threads :                                112

The output (if any) follows:



PS:

Read file <72.err> for stderr output of this job.

